src/TimeIntegrator.C

```diff
@@ -420,6 +423,43 @@ TimeIntegrator::provide_mean_norm()
       << std::setprecision(6) << timeStepCount_ << " " << currentTime_ << std::endl;
 }
 
+//--------------------------------------------------------------------------
+void
+TimeIntegrator::interstep_updates(int nonLinearIterationIndex)
+{
+  // hard code as false for now. We want to only trigger this when there is
+  // FSI so we need to add an indicator for fsi
+  const bool updateGeomInsideNL = nonLinearIterationIndex > 0 && false;
```

Comment:
Is the goal for the `nonlineariterationindex` just to not to do the "pre-prolog" and "pre-epilog" work twice at the first nonlinear iteration?

Reply:
yeah